brainfuck/bf.zig

```diff
@@ -0,0 +1,189 @@
+const std = @import("std");
+
+const OpType = enum {
+    INC,
+    MOVE,
+    LOOP,
+    PRINT,
+};
+
+const Printer = struct {
+    stdout: std.fs.File.Writer = std.io.getStdOut().writer(),
```

Comment:
It is possible to initialize the writer only if self.quiet is false? (Maybe some kind of lazy initialization). It's not like it creates some big overhead, however it would be nice to avoid any unnecessary operations.

Reply:
Will fix.

Reply:
Now initialized only when quiet is false.